cms/envs/aws.py

```diff
@@ -43,6 +43,7 @@
 
 EMAIL_BACKEND = 'django_ses.SESBackend'
 SESSION_ENGINE = 'django.contrib.sessions.backends.cache'
+DEFAULT_FILE_STORAGE = 'storages.backends.s3boto.S3BotoStorage'
```

Comment:
Ah yes, thanks for catching that, this was a mistake during my most recent rebase.